api/tools/src/main/java/org/pmiops/workbench/tools/ConfigLoader.java

```diff
@@ -52,7 +52,7 @@ public CommandLineRunner run(ConfigDao configDao) {
       if (marshalledDiff.size() > 0) {
         log.info("Configuration doesn't match WorkbenchConfig format; see diff.");
         log.info(marshalledDiff.toString());
-        return;
+        System.exit(1);
```

Comment:
Without this, scripts proceed even if your config didn't load correctly, which becomes a confusing bug to track down.

Reply:
Thanks for fixing this!